cfn/cfn.json

```diff
@@ -227,114 +98,82 @@
-                    {
-                        "PolicyName": "GetTeamKeysPolicy",
-                        "PolicyDocument": {
-                            "Statement": [
-                                {
-                                    "Effect": "Allow",
-                                    "Action": [
-                                        "s3:GetObject"
-                                    ],
-                                    "Resource": [
-                                        "arn:aws:s3:::github-team-keys/*"
-                                    ]
-                                },
-                                {
-                                    "Effect": "Allow",
-                                    "Action": [
-                                        "s3:ListBucket"
-                                    ],
-                                    "Resource": "arn:aws:s3:::github-team-keys"
-                                },
-                                {
-                                    "Effect": "Allow",
-                                    "Action": [
-                                        "ec2:DescribeTags"
-                                    ],
-                                    "Resource": "*"
-                                },
-                                {
-                                    "Effect": "Allow",
-                                    "Action": [
-                                        "s3:HeadObject",
-                                        "s3:GetObject",
-                                        "s3:GetObjectAcl"
-                                    ],
-                                    "Resource": "arn:aws:s3:::gnm-multimedia-rr-deployables/*"
-                                }
+                            "Action": [
+                                "s3:GetObject"
+                            ],
+                            "Resource": [
+                                "arn:aws:s3:::github-team-keys/*"
                             ]
-                        }
+                        }, {
+                            "Effect": "Allow",
+                            "Action": [
+                                "s3:ListBucket"
+                            ],
+                            "Resource": "arn:aws:s3:::github-team-keys"
+                        }, {
+                            "Effect": "Allow",
+                            "Action": [
+                                "ec2:DescribeTags"
+                            ],
+                            "Resource": "*"
+                        }, {
+                            "Effect": "Allow",
+                            "Action": [
+                                "s3:HeadObject",
+                                "s3:GetObject",
+                                "s3:GetObjectAcl"
+                            ],
+                            "Resource": "arn:aws:s3:::gnm-multimedia-rr-deployables/*"
```

Comment:
very minor but this isn't about getting github keys so probably should be moved into its own policy for clarity.